recipe/conda_build_config.yaml

```diff
@@ -1,6 +1,9 @@
-channel_targets:
-  - conda-forge matplotlib_rc
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - "10.12"                # [osx and x86_64]
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - 10.12                  # [osx and x86_64]
\ No newline at end of file
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+  - vs                         # [win]
+c_stdlib_version:              # [unix]
+  - 2.12                       # [linux and x86_64]
+  - 2.17                       # [linux and not x86_64]
+  - 10.13                      # [osx and x86_64]
+  - 11.0                       # [osx and arm64]
```

Comment:
Thanks Tom! 🙏

Think these should be taken care of for us, we just need to re-render

Reply:
should I drop this file then?  I only half understand what I am doing right now...

Reply:
Yep let's drop it 🙂